modules/experimental/src/gltf/gltf-material-parser.ts

```diff
@@ -1,4 +1,4 @@
-import {Device, log} from '@luma.gl/api';
+import {Device, log, Texture} from '@luma.gl/api';
```

Comment:
Nit: I think we could `import type {Device, Texture}` as we are not instantiating them ...